reference/tax_provider.yml

```diff
@@ -661,12 +661,12 @@ components:
           $ref: '#/components/schemas/TaxClass'
         tax_exempt:
           type: boolean
-          description: Flag whether or not this item is always tax exempt. For example, gift certificate purchases and order-level refunds are tax exempt. Tax exempt items are included in the request for auditing purposes.
+          description: Flag whether or not this item is always tax-exempt. For example, gift certificate purchases and order-level refunds are tax-exempt. Tax-exempt items are included in the request for auditing purposes.
           default: false
         type:
           type: string
           description: |-
-            The type of line item this request represents. This will depend on the items position in the request hirearchy, for example a collection of items (which may or may not also have wrapping attached) are contained within the document request. Seperately, each document request has a single shipping and a single handling line item (to capture these values).
+            The type of line item this request represents. This will depend on the items position in the request hierarchy, for example, a collection of items (which may or may not also have wrapping attached) are contained within the document request. Separately, each document request has a single shipping and a single handling line item (to capture these values).
```

Comment:
```suggestion
            "The type of line item this request represents. This will depend on the item's position in the request hierarchy. For example, the document request contains a collection of items (which may or may not also have wrapping attached). Separately, each document request has a single shipping and a single handling line item (to capture these values)."
```